src/NetworkDynamics.jl

```diff
@@ -36,27 +36,35 @@ constructors.
 function collect_ve_info(vertices!, edges!, graph)
     if vertices! isa Array
         @assert length(vertices!) == nv(graph)
-        v_dims = [v.dim for v in vertices!]
-        symbols_v = [Symbol(vertices![i].sym[j],"_",i) for i in 1:length(vertices!) for j in 1:v_dims[i]]
+        v_dims = Int64[v.dim for v in vertices!]
+        symbols_v = Symbol[Symbol(vertices![i].sym[j],"_",i)
```

Comment:
```suggestion
        symbols_v = [Symbol(vertices![i].sym[j],"_",i)
```
This is a bit redundant. The `Symbol` constructor can be inferred to create symbols (even for empty edges `symbols_e isa Symbol[]` before this pr. But doesn't hurt either 🤷‍♂️